schemas/layouts/layouts.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Layout",
+  "description": "A layout in Foxglove",
+  "$defs": {
```

Comment:
should these defs be after the properties so a user can learn about the layout object structure first and then go to the specific things they want?

Reply:
Done!